app/helpers/wpcc/your_results_page_helper.rb

```diff
@@ -0,0 +1,9 @@
+module Wpcc
+  module YourResultsPageHelper
+    TAX_RELIEF_THRESHOLD_RATE = 11_500
+
+    def earning_below_tax_relief_threshold?(salary)
+      salary.to_i < TAX_RELIEF_THRESHOLD_RATE
+    end
+  end
+end
```

Comment:
Does results presenter has the salary information? Shall we move this method to your results presenter? I just want to signal this different approaches to present information (which both are good) but I think having two patterns will confuse everyone. Want to know what @margOnline and @willhall88 thinks about it.